demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -337,3 +342,69 @@ def parse_docker_image(self, docker_image):
                 return 'demisto/python', self.get_docker_image_latest_tag('demisto/python', None)
             else:
                 return 'demisto/python3', self.get_docker_image_latest_tag('demisto/python3', None)
+
+    @staticmethod
+    def get_docker_image_latest_tag_from_iron_bank_request(docker_image_name):
+        """
+        Get the latest tag for a docker image by request to Iron Bank Repo.
+        Args:
+            docker_image_name: The docker image name.
+
+        Returns:
+            The latest tag for the docker image.
+        """
+        project_name = docker_image_name.replace('demisto/', '')
+        api_url = 'https://repo1.dso.mil/api/v4/projects/dsop%2Fopensource%2Fpalo-alto-networks%2Fdemisto%2F'
```

Comment:
does this URL is bound to chagne/constant? (v4), also - might we use it in another future validation? I suggest in any case move it to `constnats`